regulations/templates/regulations/tree/preamble_intro.html

```diff
@@ -31,7 +31,11 @@
         EST.</h5>
       Rule published {{ meta.publication|date:"F j, Y"}}
     </div>
-    {% comment %}Jump links will go here{% endcomment %}
+    {% if meta.supporting_documents %}
+    <div>
```

Comment:
Can do. I also assume we'll need to modify the markup when making the final pass of styling